sirepo/server.py

```diff
@@ -122,7 +122,13 @@ async def api_deleteSimulation(self):
     @sirepo.quest.Spec(
         "require_user", sid="SimId optional", filename="SimFileName", sim_data="SimData"
     )
-    async def api_downloadFile(self, simulation_type, simulation_id, filename):
+    async def api_downloadFile(self, simulation_type, filename):
+        return await self.api_downloadLibFile(simulation_type, filename)
+
+    @sirepo.quest.Spec(
+        "require_user", sid="SimId optional", filename="SimFileName", sim_data="SimData"
```

Comment:
sid is not a parameter. I don't think sim_data is a parameter either.

Reply:
[7db1329](https://github.com/radiasoft/sirepo/pull/6348/commits/7db1329e8705ff5552bb76e6feb6424e33592fc9)